SharedSources/VLCMetadata.m

```diff
@@ -61,6 +60,9 @@ - (void)updateMetadataFromMedia:(VLCMLMedia *)media mediaPlayer:(VLCMediaPlayer*
     }
     [self updatePlaybackRate:mediaPlayer];
```

Comment:
I'm not sure but do we need to `updatePlaybackRate` if the passcode lock is enabled? 🤔

Reply:
tbh I don't think so but I would not move the check above that line because it's more readable / logical to just have it above populateInfoCenter.